packages/ns-api/README.md

```diff
@@ -5343,6 +5343,124 @@ Response example:
 {"result": "success"}
 ```
 
+## ns.threatshield
+
+Manage banip configuration.
+
+### list-blocklist
+
+List current blocklist:
+```
+api-cli ns.threatshield list-blocklist
+```
+
+Response example:
+```json
+{
+  "data": [
+    {
+      "name": "yoroimallvl1",
+      "type": "enterprise",
+      "enabled": false,
+      "confidence": 10
+    },
+    {
+      "name": "yoroimallvl2",
+      "type": "enterprise",
+      "enabled": false,
+      "confidence": 8
+    }
+  ]
+}
+```
+
+Fields:
+- type can be `enterprise`, `community` or `warning`,  the warning type is when the list is present in UCI configuration but it's not a supported feed
+- confidence can be `-1` if the value is not available
+
+
```

Comment:
`edit-blocklist` documentation is missing. I'd change its interface (see other comment)